presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -62,14 +62,26 @@ export class QueryListItem extends React.Component {
             }
         }
 
+        const maxLines = 7;
+        const maxWidthPerLine = 75;
+        let lineCount = 0;
         let formattedQueryText = "";
 
         for (let i = 0; i < lines.length; i++) {
             const trimmedLine = lines[i].substring(minLeadingWhitespace).replace(/\s+$/g, '');
+            lineCount += Math.ceil(trimmedLine.length / maxWidthPerLine);
 
             if (trimmedLine.length > 0) {
                 formattedQueryText += trimmedLine;
 
+                if (formattedQueryText.length >= 300) {
```

Comment:
This magic number is used twice(here and line 91). Let's define a const variable for this.

Reply:
done